crates/hotshot/types/src/data.rs

```diff
@@ -797,7 +798,68 @@ pub struct QuorumProposal2<TYPES: NodeType> {
 
     /// The light client state update certificate for the next epoch.
     /// This is required for the epoch root.
-    pub state_cert: Option<LightClientStateUpdateCertificate<TYPES>>,
+    pub state_cert: Option<LightClientStateUpdateCertificateV2<TYPES>>,
+}
+
+/// Proposal to append a block.
```

Comment:
Please add some documentation as to why we keep this legacy type around. Should give some idea when to use it over other similar types.